<launch>


	<param name="use_sim_time" value="true"/>

	<param name="initial_pose_x" value="0.0" />
	<param name="initial_pose_y" value="0.0" />
	<param name="initial_pose_theta" value="0.0" />

	<node pkg="final_pkg" type="pub_velocities" name="pub_velocities" />
	<node pkg="final_pkg" type="pub_odometry" name="pub_odometry"/>

	<node pkg="tf" type="static_transform_publisher" name="world_odom_transform_broadcaster" args="-0.832142115 0.42636168 0.32472017404 0.0221652016044 -0.0104456143454 -0.533567428589 -0.845402538776 world odom 100" />

</launch>